modules/jboss.container.jolokia/jkube-1.7.2/module.yaml

```diff
@@ -3,7 +3,7 @@
 schema_version: 1
 
 name: jboss.container.jolokia
-version: 'jkube-1.7.2'
+version: 'jkube-2.0.0'
```

Comment:
This is not OK, a new module version is needed. I'll proceed to add some changes on top of yours

Reply:
That will be great